src/middlewared/middlewared/plugins/service_monitor.py

```diff
@@ -82,31 +85,36 @@ def isEnabled(self, service):
         return enabled
 
     @private
-    def tryConnect(self, host, port):
+    def tryConnect(self, host, port, fnldap):
```

Comment:
Better to declare it as: `def tryConnect(self, host, port, fnldap = None):`

Reply:
Actually, as we only call `get_ldap_servers` we don't need all this code around `fnldap`, see below.